node/src/components/small_network.rs

```diff
@@ -760,13 +755,16 @@ where
             Message::Handshake {
                 network_name,
                 public_address,
+                protocol_version,
             } => {
-                if network_name != self.network_name {
+                if network_name != self.chain_info.network_name {
```

Comment:
Hm, so we are including the protocol version in the handshake but not using it? Is that intentional?

Reply:
Yes, it is right now. It is for future use.